src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -262,6 +275,47 @@ export default {
 
             this.time = this.timeFormat.format(datum);
             this.imageUrl = this.imageFormat.format(datum);
+        },
+        selectedImageIndex() {
+            return this.imageHistory.findIndex(image => image.selected);
+        },
+        setSelectedByIndex(index) {
+            this.setSelectedImage(this.imageHistory[index]);
+        },
+        nextImage() {
+            let index = this.selectedImageIndex();
+            this.setSelectedByIndex(++index);
+            if (index === this.imageHistory.length - 1) {
+                this.paused(false);
+            }
+        },
+        prevImage() {
+            let index = this.selectedImageIndex();
+            if (index === -1) {
+                this.setSelectedByIndex(this.imageHistory.length - 2);
+            } else {
+                this.setSelectedByIndex(--index);
+            }
+        },
+        isNextDisabled() {
+            let disabled = false;
+            let index = this.selectedImageIndex();
+
+            if (index === -1 || index === this.imageHistory.length - 1) {
+                disabled = true;
+            }
+
+            return disabled;
+        },
+        isPrevDisabled() {
+            let disabled = false;
+            let index = this.selectedImageIndex();
+
+            if (index === 0) {
```

Comment:
Also need to disable this when `imageHistory` length is `0` or `1`

Reply:
added a check for less than 2!